package.json

```diff
@@ -70,8 +75,9 @@
     "build": "node ./scripts/build.js",
     "test": "cross-env TZ=utc jest",
     "eject": "react-scripts eject",
-    "lint": "eslint src",
-    "server": "node server.js",
+    "lint": "eslint src server",
+    "server": "ts-node -T -P server/tsconfig.json server/index.ts",
+    "build:server": "tsc -p server/",
```

Comment:
Why do we need to compile the ts files if we are already using ts-node?

Reply:
I added this script so that we could check for any errors during our Circle CI builds. I suppose we only need to perform type checking, not compilation. I'll change the script name to better describe this and also add `--noEmit`.

Reply:
I see. When the app is run in dev mode with `npm start`, the api requests made to `/token` will be [proxied](https://github.com/twilio/twilio-video-app-react/blob/master/package.json#L104) to `localhost:8081/token`. So from that app's point of view, everything is on the same origin in dev mode. So I don't think we'll need them for that. But if we ever need them in the future I'm happy to add them in.